### PR TITLE
Update OBS to Rails 5.2

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/models/kiwi/preference.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class Kiwi::Preference < ApplicationRecord
   belongs_to :image, inverse_of: :preference
 
   enum type_image: [:btrfs, :clicfs, :cpio, :docker, :ext2, :ext3, :ext4, :iso, :lxc, :oem,
-                    :product, :pxe, :reiserfs, :split, :squashfs, :tbz, :vmx, :xfs, :zfs]
+                    :product, :pxe, :reiserfs, :split, :squashfs, :tbz, :vmx, :xfs, :zfs], _prefix: :image_type
```
Comment (hennevogel) [inline src/api/app/models/kiwi/preference.rb]: Hm so `type_image.image_type_split?`

File: .rubocop.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@ AllCops:
     - 'src/api/test/fixtures/backend/**/*'
     - 'src/api/files/*'
     - 'src/backend/**/*'
+    - 'src/api/db/data_schema.rb'
```
Comment (hennevogel) [inline .rubocop.yml]: You forgot to drop this commit

File: src/api/app/models/kiwi/preference.rb (hunk 1 above)
Comment (bgeuken) [reply src/api/app/models/kiwi/preference.rb]: Right

File: .rubocop.yml (hunk 2 above)
Comment (bgeuken) [reply .rubocop.yml]: Dropped it now

**Pull request merged**